include/wil/resource.h

```diff
@@ -6560,6 +6560,8 @@ namespace wil
         return{};
     }
 
+//! WDM version of EX_PUSH_LOCK is available starting with Windows 10 1809
+#if NTDDI_VERSION >= NTDDI_WIN10_RS5

```

Comment:
Thanks - nitpicking, please add parentheses to follow the pattern elsewhere in the repo:

```c
#if (NTDDI_VERSION >= NTDDI_WIN10_RS5)
```

Reply:
Fixed

Reply:
I used the information from the MSDN documentation. But it seems that it is not quite correct: the `Ex*PushLock` API, apparently appeared earlier than indicated [here](https://github.com/MicrosoftDocs/windows-driver-docs-ddi/blob/staging/wdk-ddi-src/content/wdm/nf-wdm-exinitializepushlock.md).
Based on the contents of the wdm.h header file, I came to the conclusion that this API appeared starting with Windows 8.1
```cpp
#ifndef EX_NO_PUSH_LOCKS
#if (NTDDI_VERSION < NTDDI_WINBLUE)
#ifndef EX_LEGACY_PUSH_LOCKS
#define EX_LEGACY_PUSH_LOCKS
#endif
#endif // (NTDDI_VERSION < NTDDI_WINBLUE)
#endif // EX_NO_PUSH_LOCKS
```